### DISCUSSION TITLE
[help] Weird problem with tar_outdated

### DISCUSSION THREAD

**Discussion by ksvanhorn** [ropensci/targets#1195] [Help]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

A few days ago targets started giving me the following error when I do tar_outdated() on certain targets:

Error:
! Error running targets::tar_outdated()
  Error messages: targets::tar_meta(fields = error, complete_only = TRUE)
  Debugging guide: https://books.ropensci.org/targets/debugging.html
  How to ask for help: https://books.ropensci.org/targets/help.html
  Last error: no applicable method for 'store_class_format' applied to an object of class "c('NA', 'character')"

That last line is pretty strange. Any idea what can cause this?

**Comment by wlandau** [comment]
Sounds like you may be running a newer version of `targets` on a project you began with an older version. If that's the case, you could downgrade to that old version, or you could call `tar_destroy()` if the current pipeline results are not important. If the problem persists, please post a reprex: https://books.ropensci.org/targets/help.html#write-a-reprex
